pkgs/applications/misc/foot/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv, fetchgit, fcft, freetype, pixman, libxkbcommon, fontconfig, wayland
+, meson, ninja, ncurses, scdoc, tllist, wayland-protocols, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "foot";
+  version = "1.4.4";
+
+  src = fetchgit {
+    url = "https://codeberg.org/dnkl/foot.git";
+    rev = "${version}";
+    sha256 = "1cr4sz075v18clh8nlvgyxlbvfkhbsg0qrqgnclip5rwa24ry1lg";
+  };
+
+  nativeBuildInputs = [ meson ninja ncurses scdoc tllist wayland-protocols ];
```

Note that `pkg-config` moved here.

```suggestion
  nativeBuildInputs = [
    meson
    ninja
    ncurses
    scdoc
    tllist
    wayland-protocols
    pkg-config
  ];
```

---

whoops, pkg-config is in the wrong place indeed